test-lit/unit/lit.local.cfg

```diff
@@ -1 +1,2 @@
-config.suffixes = ['.f90', '.F90']
+config.suffixes = [ '.f', '.F', '.f18', '.F18', '.f90', '.F90', '.f95', '.F95', '.ff', '.ff18', '.ff90', '.ff95']
```

Comment:
I think it would be better to just set the global config.suffixes to all the fortran input file extensions F18 know about (earlier comment of mine) and then you don't need to set this here and the lit.local.cfg file can be removed.

Reply:
This file was removed

Reply:
Done!